packages/tasks/src/model-libraries.ts

```diff
@@ -222,6 +222,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"checkpoints/byt5_model.pt"`,
 	},
+	"gpt-neox": {
```

Comment:
This PR is actually a bit problematic. As we have 4k models with `gpt-neox` tag (automatically determined due to `model_type` in `config.json`), this will lead to some issues. I wonder if we could explore a different name `gpt-neox-original` or something like that for the tag (to be added to all repos that can run inference with the `gpt-neox` library). WDYT?

Reply:
I'm not deeply attached to either stylism. If we can rename 33 models and use a slightly different tag that's fine with me. I think the core question is if y'all like style consistency more than you dislike making a very large number of small edits. That said, if most of these models are getting automatically tagged there may need to be a CI that runs on new models to correct the tag each time. Would it be possible to hack the GPT-NeoX _model class definition_ so that using `gpt_neox` induces the metadata entry `gpt-neox`? Or is that too wonky of a patch? I assume changing the config name is non-viable at this point in time.

As a side note, the above two comments are contradictory. To be clear, it looks to me that gpt**-**neox is the small one and gpt**_**neox is the big one.